app/components/agenda/agenda-detail/sidebar-item.js

```diff
@@ -102,9 +102,17 @@ export default class SidebarItem extends Component {
   @action
   conditionallyScrollIntoView(element) {
     if (this.args.isActive) {
-      element.scrollIntoView({
-        behavior: 'smooth', block: 'center',
-      });
+      const container = document.getElementById('agendadetail-sidebar-container');
```

Comment:
Mocht er een betere manier zijn om dit container element aan te spreken, laat gerust iets weten bij een review.

Reply:
Seems fine to me, there might be a better way using the `ember-render-modifiers` but that is probably not worth the effort.